jbpm-system-test/jbpm-unmarshall-ejb-timers-test/src/test/java/org/kie/samples/integration/UnmarshallEJBTimerSystemTest.java

```diff
@@ -0,0 +1,220 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.samples.integration;
+
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.kie.samples.integration.testcontainers.KieServerContainer;
+import org.kie.server.api.marshalling.MarshallingFormat;
+import org.kie.server.api.model.KieContainerResource;
+import org.kie.server.api.model.ReleaseId;
+import org.kie.server.client.KieServicesClient;
+import org.kie.server.client.KieServicesConfiguration;
+import org.kie.server.client.KieServicesFactory;
+import org.kie.server.client.ProcessServicesClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.BindMode;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.github.dockerjava.api.DockerClient;
+import com.zaxxer.hikari.HikariConfig;
+import com.zaxxer.hikari.HikariDataSource;
+
+@Testcontainers(disabledWithoutDocker=true)
+public class UnmarshallEJBTimerSystemTest {
+    
+    public static final String SELECT_PARTITION_NAME_FROM_JBOSS_EJB_TIMER = "select partition_name from jboss_ejb_timer";
+    public static final String PREFIX_CLI_PATH = "src/test/resources/etc/jbpm-custom-";
+    public static final String SELECT_COUNT_FROM_JBOSS_EJB_TIMER = "select count(*) from jboss_ejb_timer";
+    public static final String ARTIFACT_ID = "unmarshall-ejb-sample";
+    public static final String GROUP_ID = "org.kie.server.testing";
+    public static final String VERSION = "1.0.0";
+    public static final String ALIAS = "-alias";
+    
+    public static final String DEFAULT_USER = "kieserver";
+    public static final String DEFAULT_PASSWORD = "kieserver1!";
+
+    public static String containerId = GROUP_ID+":"+ARTIFACT_ID+":"+VERSION;
+
+    private static Logger logger = LoggerFactory.getLogger(UnmarshallEJBTimerSystemTest.class);
+    
+    private static Map<String, String> args = new HashMap<>();
+
+    static {
+        args.put("IMAGE_NAME", System.getProperty("org.kie.samples.image"));
+        args.put("START_SCRIPT", System.getProperty("org.kie.samples.script"));
+        args.put("SERVER", System.getProperty("org.kie.samples.server"));
```

Comment:
maybe some default system property values? wdyt?

Reply:
not needed, because image version will be changed every release

Reply:
not needed, because image version will be changed every release